catalog-rest-service/src/main/resources/json/schema/entity/services/connections/database/mssqlConnection.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/services/connections/database/mssqlConnection.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "MssqlConnection",
+  "description": "Mssql Database Connection Config",
+  "type": "object",
+  "javaType": "org.openmetadata.catalog.services.connections.database.MssqlConnection",
+  "properties": {
+    "username": {
+      "description": "username to connect  to the MsSQL. This user should have privileges to read all the metadata in MsSQL.",
+      "type": "string"
+    },
+    "password": {
+      "description": "password to connect  to the MsSQL.",
+      "type": "string"
+    },
+    "hostPort": {
+      "description": "Host and port of the MsSQL.",
+      "type": "string"
+    },
+    "database": {
+      "description": "Database of the data source. This is optional parameter, if you would like to restrict the metadata reading to a single database. When left blank , OpenMetadata Ingestion attempts to scan all the databases in MsSQL.",
+      "type": "string"
+    },
+    "usePymssql": {
+      "description": "configure pymssql as the driver to connect to MSSQL",
+      "type": "boolean",
+      "default": "false"
+    },
+    "usePyodbc": {
+      "description": "configure pyodbc as the driver to connect to MSSQL",
+      "type": "boolean",
+      "default": "false"
+    },
+    "mssqlURI": {
```

Comment:
maybe a stupid question, why are we do we need the URI if we have the `hostPort`?

Reply:
@ayush-shah why we need this URI?